bot/cogs/moderation/management.py

```diff
@@ -228,6 +228,10 @@ def infraction_to_string(self, infraction: utils.Infraction) -> str:
         actor = guild.get_member(actor_id)
         active = infraction["active"]
         user_id = infraction["user"]
+        if guild.get_member(user_id) is not None:  # If the user is still in the server
+            user_nickname = guild.get_member(user_id).nick  # If nickname different from username show nickname
+        else:
+            user_nickname = None
```

Comment:
Can be set to `get_user()` instead of `None` to avoid doing `x or y` later. Keep in mind `get_user()` also returns `None` so it'd be good to set it to an empty string in such case to avoid displaying `"None"` as the username.

Reply:
I am not sure but it's quite likely that if a user is not in the guild then they wont be in the cache either and thus `get_user()` won't find them. Could use `fetch_user()` instead, downside being that makes an API call. However, I think that's OK. Anyone else have thoughts on that?